### PR TITLE
ERFC-740-instruction file adding

### PR DESCRIPTION
**Pull request by omorgunova-pl**

### CODE REVIEW

**Comment by omorgunova-pl** [comment]
I have read the CLA Document and I hereby sign the CLA

File: instruction.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        "face_name": "lisan"
+        }
+    ]
+    }
+]
+
+The following JavaScript code example allows a user to choose an image and send it to the server. The server side script will recieve the file, and then unencoded it using CompreFace API.
+To run the JavaScript example just load the source code into an browser console.
+
+async function UploadPhoto()
```
Comment (pospielov) [inline instruction.md:50]:
```suggestion
async function SaveNewImageToFaceCollection()
```

File: instruction.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+{
+let user = { face_name:'john'};
+let formData = new FormData();
+let photo = document.getElementById("fileDropRef").files[0];
+
+    formData.append("photo", photo);
+    formData.append("user", JSON.stringify(user));
+
+    try {
+       let r = await fetch('http://exadel.ua:8000/api/v1/faces/?subject=<user.face_name>', {method: "POST", body: formData});
```
Comment (pospielov) [inline instruction.md:60]: exadel.ua -> localhost

File: instruction.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+{
+"similarity": 0.99593,
+"face_name": "lisan"
+}
+]
+}
+]
+
+The following JavaScript code example allows to add new face to Face Collection.
+To run the JavaScript example just load the source code into an browser console.
```
Comment (pospielov) [inline instruction.md:48]: Delete `To run the JavaScript example just load the source code into an browser console.`

File: instruction.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+    try {
+       let r = await fetch('http://localhost:8000/api/v1/faces/?subject=`${name}`', {method: "POST", body: formData});
+     } catch(e) {
+       console.log('Houston, we have a problem...:', e);
+    }
+
+}
+
+To recognize a face, you should use the following:
```
Comment (pospielov) [inline instruction.md:66]:
```suggestion
This function sends image to our server and shows result in text area:
```

File: instruction.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+## Project setup and usage
```
Comment (pospielov) [inline instruction.md:1]: Delete this paragraph

Comment (pospielov) [inline instruction.md:1]: Please make this file more prettier. Here is how it looks like now:
https://github.com/exadel-inc/CompreFace/blob/EFRC-740-instruction-file-writing/instruction.md

File: instruction.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+]
+```
+
+The following JavaScript code example allows to add new face to Face Collection.
+
+```
+
+async function saveNewImageToFaceCollection()
+{
+let name = encodeURIComponent('John');
```
Comment (pospielov) [inline instruction.md:51]: Please add indent to this and next three lines

File: instruction.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+This function sends image to our server and shows result in text area:
+
+```
+
+function recognizeFace(input) {
+
+  async function getData() {
+    let response = await fetch('http://localhost:8000/api/v1/recognize')
+    let data = await response.json()
+      return data
```
Comment (pospielov) [inline instruction.md:76]: Please align this line with previous

File: instruction.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+  };
+
+  let result = Promise.resolve(response)
+    result.then(data => {
+    array = data
```
Comment (pospielov) [inline instruction.md:81]: Why do we need array variable? can we just use `JSON.stringify(data)` on the next line?

File: instruction.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+Step 2. Create an application (left section) with "Create" link at the bottom of the page. An application is where you can create and manage your face collections.
+
+Step 3. Enter you application with double click on the name of the application. Here you will have two possibilities. The first one is to add new users to your application and manage permissions (Owner and Administrator roles already have access to any application without invite, user role doesn't.) The second one is to create face collections.
+
+Step 4. After creating new collection, it appears at the Face Collections List created within the application with an appropriate name and [API key](#api-key). The user has the possibility to add new Face or to test the existing one (three dots on right side and click "test" link). This option will lead the user to Test Face Collection page, where is the drag&drop to upload image with face to recognize. We recommend an image size no higher than 5MB, as it could slow down the request process. Supported image formats are JPEG/PNG/JPG/ICO/BMP/GIF/TIF/TIFF format.
```
Comment (pospielov) [inline instruction.md:13]: [API key](#api-key) - no link anymore

File: instruction.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+Step 5. Upload your photo and let Face Recognition system compare faces. When you have face contour detection enabled (green borders around the face). These points represent the shape of the feature. API requests within the solution use RESTful API, and backend data collection. [Read more about API](./README.md) With Face Recognition system APIs you can add Face Recognition capabilities using simple API Calls.
```
Comment (pospielov) [inline instruction.md:15]: [Read more about API](./README.md) - here you can give link to this api description: https://github.com/exadel-inc/CompreFace#rest-api-description

Comment (pospielov) [inline instruction.md:15]: " When you have face contour detection enabled (green borders around the face). These points represent the shape of the feature." - what is it about ? What is `face contour detection`?
"API requests within the solution use RESTful API, and backend data collection." - what is `backend data collection`?

File: instruction.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+The following result Json illustrates how these points map to a face, where
+
+1. face_name - name of the trained model
```
Comment (pospielov) [inline instruction.md:19]: we renamed `face_name` to `subject`
and this is not the name of model, this is the person identificator

File: instruction.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+2. similarity (probability) - set the percent match of a detected face with a reference photo
```
Comment (pospielov) [inline instruction.md:20]: `probability` gives the confidence that this is a face.
`similarity` gives the confidence that this is the found subject

File: instruction.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+3. x,y - coordinates of the frame containing the face
```
Comment (pospielov) [inline instruction.md:21]: we don't have `x` and `y`

File: instruction.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+4. Box coordinates are available set values for key in format x_min,x_max,y_min,y_max and the API will provided analysis for just one face
```
Comment (pospielov) [inline instruction.md:22]: Sorry, but I didn't understand what you wanted to tell here, please contact me directly and explain :)

File: instruction.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+Face Recognition system is a way of recognizing a human face through technology. A facial recognition system uses facial features from a photograph and compares the information with a database of known faces to find a match.
+
+So how does facial recognition work?
+
+Step 1. You need to sign up to the system (First user in CompareFace admin has Owner role, but it is possible to change the role) and then LogIn with created account or just use the existing one. After that system redirects you to the main page.
+
+Step 2. Create an application (left section) with "Create" link at the bottom of the page. An application is where you can create and manage your face collections.
+
+Step 3. Enter you application with double click on the name of the application. Here you will have two possibilities. The first one is to add new users to your application and manage permissions (Owner and Administrator roles already have access to any application without invite, user role doesn't.) The second one is to create face collections.
```
Comment (pospielov) [inline instruction.md:11]: `Owner and Administrator roles` - let's call it "Global Owner and Administrator roles"

**Pull request closed without merging**